src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/client/BuildPayloadBuilder.java

```diff
@@ -26,39 +24,47 @@
      * @return an assembled Builds payload
      */
     public static Builds getBuildPayload(
-            final RunWrapper buildWrapper, final Set<String> issueKeys) {
+            final State jiraState, final RunWrapper buildWrapper, final Set<String> issueKeys) {
```

Comment:
Passing the `jiraState` into this method instead of calculating it within the method because I needed the state before this method was even called to assert against the state in tests.